src/webextension/list/common.js

```diff
@@ -1,7 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is a "six of one, half dozen of the other" kind of thing, but I would have expected `flattenItem` / `unflattenItem` to move here.